src/pages/sign-up/SignUp.jsx

```diff
@@ -0,0 +1,112 @@
+import React, { useState } from 'react'
+import PropTypes from 'prop-types'
+import { useForm, FormProvider } from 'react-hook-form'
+
+import WelcomePage from '../welcome-page/WelcomePage'
+import { CloseIcon } from '../../components/icons/icons'
+import SignInAndUpHeading from '../../components/UI/sign-in-up-heading/SignInAndUpHeading'
+import Input from '../../components/UI/input/Input'
+
+import styles from './SignUp.module.scss'
+import SocialMediaSignIn from '../../components/UI/social-media-sign-in/SocialMediaSignIn'
+
+const SignUp = ({ showSignUp, setShowSignUp, showSignIn, setShowSignIn }) => {
+  const [welcome, setWelcome] = useState(false)
+
+  const methods = useForm()
+  const onFormSubmit = data => {
+    console.log(data)
```

Comment:
remove please

Reply:
I am not sure if want you want me to do is to remove line 16, 17 and 18, because line 16 is necessary for the react-hook-form to work as it gives me access to methods like the handleSubmit which is used to submit the form, but I assumed its the console.log you wanted me to remove so I removed just that.